docker/.env

```
@@ -259,6 +272,27 @@ MG_MQTT_ADAPTER_INSTANCE=
 MG_MQTT_ADAPTER_INSTANCE_ID=
 MG_MQTT_ADAPTER_ES_DB=0
 
+## MPROXY MQTT
+MG_MQTT_ADAPTER_ADDRESS=:1883
+MG_MQTT_ADAPTER_TARGET=${MG_MQTT_BROKER_TYPE}:1883
+MG_MQTT_ADAPTER_CERT_FILE=/etc/ssl/certs/magistrala-server.crt
+MG_MQTT_ADAPTER_KEY_FILE=/etc/ssl/certs/magistrala-server.key
+MG_MQTT_ADAPTER_SERVER_CA_FILE=/etc/ssl/certs/ca.crt
+# MG_MQTT_ADAPTER_CLIENT_CA_FILE=etc/ssl/certs/ca.crt
+# MG_MQTT_ADAPTER_CERT_VERIFICATION_METHODS=ocsp
+# MG_MQTT_ADAPTER_OCSP_RESPONDER_URL=http://localhost:8080/ocsp
+
+## MPROXY MQTT WS 
+MG_MQTT_WS_ADAPTER_ADDRESS=:8080
+MG_MQTT_WS_ADAPTER_TARGET=ws://${MG_MQTT_BROKER_TYPE}:8080/mqtt
+MG_MQTT_WS_ADAPTER_PREFIX_PATH=/mqtt
+# MG_MQTT_WS_ADAPTER_CERT_FILE=
+# MG_MQTT_WS_ADAPTER_KEY_FILE=
+# MG_MQTT_WS_ADAPTER_SERVER_CA_FILE=etc/ssl/certs/ca.crt
+# MG_MQTT_WS_ADAPTER_CLIENT_CA_FILE=etc/ssl/certs/ca.crt
+# MG_MQTT_WS_ADAPTER_CERT_VERIFICATION_METHODS=ocsp
+# MG_MQTT_WS_ADAPTER_OCSP_RESPONDER_URL=http://localhost:8080/ocsp
+
```

We should provide the certificate location , not the docker location
Same for HTTP Adapter also. Please change in HTTP Adapter env values to certificates location
```suggestion
MG_MQTT_ADAPTER_CERT_FILE=./ssl/magistrala-server.crt
MG_MQTT_ADAPTER_KEY_FILE=./ssl/magistrala-server.key
MG_MQTT_ADAPTER_SERVER_CA_FILE=./ssl/certs/ca.crt
MG_MQTT_ADAPTER_CLIENT_CA_FILE=./ssl/certs/ca.crt
```